examples/identity.rs

```diff
@@ -0,0 +1,22 @@
+use multihash::Multihash;
+
+//! An example for how to use the "identity" hash of [`Multihash`].
+//!
+//! Identity hashing means we don't actually perform any hashing.
+//! Instead, we just store data directly in place of the "digest".
+//!
+//! [`Multihash::wrap`] returns an error in case the provided digest is too big for the available space.
+//! Make sure you construct a [`Multihash`] with a large enough buffer for your data.
+//!
+//! Typically, the way you want to use the "identity" hash is:
+//! 1. Check if your data is smaller than whatever buffer size you chose.
+//! 2. If yes, store the data inline.
+//! 3. If no, hash it make it fit into the provided buffer.
+
+fn main() {
+    let valid_identity_hash = Multihash::<64>::wrap(0, b"foobar").unwrap();
```

Comment:
What about adding a constant like `const IDENTITY_CODE: u64 = 0`, to make it a bit clearer what the magic value `0` in the `wrap()` call means.

Reply:
Sounds good. I'll link to the csv in the repository too.